src/engine/BMBtnSkillRandomBMPentaskill.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * BMBtnSkillRandomBMPentaskill: Code specific to RandomBMPentaskill
+ *
+ * @author: james
+ */
+
+/**
+ * This class currently supports the special skills of RandomBMPentaskill, which has
+ * 4 normal dice and a swing die, with 5 skills appearing a total of 13 times on various dice.
```

Comment:
That's what the button description says, or should anyway, based on what I came up with on https://github.com/buttonmen-dev/buttonmen/issues/779#issuecomment-78077615 way back when.